two_factor/views/core.py

```diff
@@ -213,7 +213,10 @@ class SetupView(IdempotentSessionWizardView):
         ('validation', DeviceValidationForm),
         ('yubikey', YubiKeyDeviceForm),
     )
+
+    show_welcome = not hasattr(settings, 'TWO_FACTOR_SKIP_WELCOME') or not settings.TWO_FACTOR_SKIP_WELCOME
```

Comment:
1. Change `False` to `None` then.
2. What's the problem of being executed at import time?